### PR TITLE
Utilize bowling clothing

### PR DESCRIPTION
**Pull request by Matt9440**
Copied the default ClothingContainer into our project and renamed it to ClothesContainer - allowing us to directly change what we need to.

Create bowling clothing assets.

Equip the bowling clothes in DressPlayer()
image
image

### CODE REVIEW

**Review by Adesii — CHANGES_REQUESTED** [review]
why did you add the ClothesContainer?
seems to be a 1:1 copy of the original

File: code/Base/ClothesContainer.cs [hunk 1]
```diff
@@ -0,0 +1,202 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Sports
+{
+	/// <summary>
+	/// Holds a collection of clothing items. Won't let you add items that aren't compatible.
+	/// </summary>
+	public class ClothesContainer
```
Comment (Adesii) [inline code/Base/ClothesContainer.cs:8]: why is this needed

**Review by Adesii — APPROVED** [review]
Looks good to me

File: code/Base/ClothesContainer.cs (hunk 1 above)
Comment (Matt9440) [reply code/Base/ClothesContainer.cs]: As discussed
